frontend/reports/service-interests/service-interests-results.component.tsx

```diff
@@ -70,4 +72,36 @@ export default function ServiceInterestsResults(props) {
       footerRows={null}
     />
   );
+
+  function getCsvOptions(): Promise<CsvOptions> {
+    const interestResults = [];
+    data.programs.map((program) => {
```

Comment:
When you're not storing the result of `.map()`, there's no point in using `.map()`. Instead, you should use `.forEach()`.

```suggestion
    data.programs.forEach((program) => {
```